alf/algorithms/rl_algorithm.py

```diff
@@ -347,13 +379,6 @@ def train_complete(self,
         valid_masks = tf.cast(
             tf.not_equal(training_info.step_type, StepType.LAST), tf.float32)
 
-        # reward shaping
-        if self._reward_shaping_fn is not None:
-            # record unshaped extrinsic rewards given by the environment
-            self.add_reward_summary("reward/raw", training_info.reward)
-            training_info = training_info._replace(
-                reward=self._reward_shaping_fn(training_info.reward))
-
```

Comment:
This "raw" reward is no longer summarized anywhere.

Reply:
Part of the reason for removing it is that I don't find a good place to do it. And part of reason is that it is not that useful.